apps/framework-cli/src/cli/commands.rs

```diff
@@ -38,10 +38,16 @@ pub enum Commands {
     // },
     /// Starts a local development environment to build your data-intensive app or service
     Dev {},
+    /// Start a remote environment for use in cloud deployments
+    Prod {},
     // Updates the redpanda cluster and clickhouse database with the latest objects
     Update {},
     // Stops development infrastructure
     Stop {},
     // Clears all temporary data and stops development infrastructure
     Clean {},
+    /// Docker related commands
+    Docker {
+        sub_method: String,
```

Comment:
Same as above. Could be an enum of the docker methods you want. Again, not a blocker just calling out